qcodes/utils/validators.py

```diff
@@ -496,9 +498,10 @@ class Arrays(Validator):
 
     validtypes = (int, float, np.integer, np.floating)
 
-    def __init__(self, min_value: numbertypes=-float("inf"),
-                 max_value: numbertypes=float("inf"),
-                 shape: TSequence[Union[int, TCallable[[], int]]]=None) -> None:
+    def __init__(self, min_value: numbertypes = -float("inf"),
+                 max_value: numbertypes = float("inf"),
+                 shape: TSequence[
+                     Union[int, TCallable[[], int]]] = None) -> None:
```

Comment:
any value in giving `Union[int, TCallable[[], int]]` a type alias, say, `shape_type`?